src/resources/Change/GeneralPartnershipResource.ts

```diff
@@ -31,7 +30,10 @@ export const GeneralPartnershipResource: ResourceIF = {
           'If you require assistance with adding a business partner please contact us.'
         ]
       }
-    }
+    },
+    nameRequestTypes: [
+      NameRequestTypes.CHANGE_OF_NAME
+    ]
```

Comment:
I defined this here so the mixin code wouldn't use the default (CHG and CNV).

Ditto SP resources below.